### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by EPashkin**
Followup of #539

cc @GuillaumeGomez , @MathieuDuponchelle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,10 +301,10 @@ mod color;
 And finally the last feature! Just run the following command (note the `-m doc` at the end):
 
 ```shell
-cargo run --release -- -c YourGirFile.toml -d ../gir-files -o the-output-directory -m doc
+cargo run --release -- -c YourGirFile.toml -d ../gir-files --doc-target-path the-output-file-name -m doc
```
Comment (GuillaumeGomez) [inline README.md]: The `-o` option isn't used for docs generation?

Comment (EPashkin) [reply README.md]: It can be used, but file name will be "vendor.md"

Comment (MathieuDuponchelle) [reply README.md]: yes, that made things a bit awkward in gstreamer

**Comment by EPashkin** [comment]
Ok, I merge this as no one against it.

**Pull request merged**